adr/query_pushdown.md

```diff
@@ -0,0 +1,42 @@
+Select queries of the form `{_id: <ObjectID>}` (value type is `ObjectID`).
+
+## Postgres
+
+If value type is not `ObjectID`, fallback to fetch entire table.
+If value type is `ObjectID` raise error (`fjson` unmarshal).
```

Comment:
I don't understand that sentence. Does it mean "If value type is `ObjectID` then raise error"?

Reply:
> I don't understand that sentence. Does it mean "If value type is `ObjectID` then raise error"?

ooops, I meant if the value type is correct but the data in the value is corrupted, the raise error. (fixing)